### ISSUE TITLE
Exception "'NoneType' object has no attribute 'qname'"

### ISSUE THREAD

**Issue by tomaszkyc** [Workiva/ixbrl-viewer#80]
When using GUI or CMD I have the same exception. Exception from CMD as example:

```
[info] loaded in 1,24 secs at 2020-01-23T14:25:40 - C:\Program Files\Arelle\ixbrl-report.xhtml
[AttributeError] [Exception] Failed to complete request:
'NoneType' object has no attribute 'qname'
['  File "Z:\\Users\\arelle\\hfdev\\arelle\\CntlrCmdLine.py", line 1020, in run\n', '  File "C:\\Users\\<my-username>\\git\\ixbrl-viewer\\iXBRLViewerPlugin\\__init__.py", line 70, in commandLineRun\n    iXBRLViewerCommandLineXbrlRun(*args, **kwargs)\n', '  File "C:\\Users\\<my-username>\\git\\ixbrl-viewer\\iXBRLViewerPlugin\\__init__.py", line 38, in iXBRLViewerCommandLineXbrlRun\n    iv = viewerBuilder.createViewer(scriptUrl=options.viewerURL)\n', '  File "C:\\Users\\<my-username>\\git\\ixbrl-viewer\\iXBRLViewerPlugin\\iXBRLViewer.py", line 252, in createViewer\n    self.addConcept(f.concept)\n', '  File "C:\\Users\\<my-username>\\git\\ixbrl-viewer\\iXBRLViewerPlugin\\iXBRLViewer.py", line 133, in addConcept\n    conceptName = self.nsmap.qname(concept.qname)\n'] - .\ixbrl-report.xhtml
PS C:\Program Files\Arelle>
```

Python version: **3.8.1**

Please help

**Comment by sagesmith-wf** [comment]
Can you please provide the filing that you are attempting to validate?

**Comment by tomaszkyc** [comment]
In this [link](https://filebin.net/c7ma9kd0p9vfaghz/bzwbk-20161231.xhtml?t=gpe9exh3) I've added document i want to convert to html with iXBRLViewerPlugin.

I think that the bug is in this line:

```
['  File "Z:\\Users\\arelle\\hfdev\\arelle\\CntlrCmdLine.py", line 1020, in run\n', '  File "C:\\Users\\<my-username>\\git\\ixbrl-viewer\\iXBRLViewerPlugin\\__init__.py", line 70, in commandLineRun\n    iXBRLViewerCommandLineXbrlRun(*args, **kwargs)\n', '  File "C:\\Users\\<my-username>\\git\\ixbrl-viewer\\iXBRLViewerPlugin\\__init__.py", line 38, in iXBRLViewerCommandLineXbrlRun\n    iv = viewerBuilder.createViewer(scriptUrl=options.viewerURL)\n', '  File "C:\\Users\\<my-username>\\git\\ixbrl-viewer\\iXBRLViewerPlugin\\iXBRLViewer.py", line 252, in createViewer\n    self.addConcept(f.concept)\n', '  File "C:\\Users\\<my-username>\\git\\ixbrl-viewer\\iXBRLViewerPlugin\\iXBRLViewer.py", line 133, in addConcept\n    conceptName = self.nsmap.qname(concept.qname)\n'] - .\ixbrl-report.xhtml
```
and it's a bug on Arelle site (because I don't have any network drive mapped to "Z" letter.

Do you need any other info?

**Comment by paulwarren-wk** [comment]
I think this got fixed in dcebb23b.

Closing this issue.
